patches/src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -1844,42 +1845,22 @@ Click to see how to issue a API key."</string>
     <!-- PreferenceScreen: Miscellaneous -->
     <string name="revanced_preference_screen_misc_title">Miscellaneous</string>
 
-    <string name="revanced_enable_debug_logging_title">Enable debug logging</string>
-    <string name="revanced_enable_debug_logging_summary_on">Debug logs are enabled.</string>
-    <string name="revanced_enable_debug_logging_summary_off">Debug logs are disabled.</string>
+    <string name="revanced_disable_quic_protocol_title">Disable QUIC protocol</string>
+    <string name="revanced_disable_quic_protocol_summary">"Disable CronetEngine's QUIC protocol."</string>
     <string name="revanced_enable_debug_buffer_logging_title">Enable debug buffer logging</string>
```

Comment:
The setting `Enable debug buffer logging` has a parent setting `Enable debug logging`.